organization/team.md

```diff
@@ -50,6 +50,12 @@ organizing Google Summer of Code for NumFOCUS.
 | Phil Elson         |                 |                |
 | Thomas Caswell     |                 |                |
 
+#### Pandas
+
+| Name                | Melange Profile | GitHub Profile |
+|---------------------|-----------------|----------------|
+| Jeff Reback         |                 |                |
```

Comment:
@rgaiacs 

pls also add: https://github.com/DevasenaInupakutika as a mentor, specically for a IO on R datasets